src/sssom/writers.py

```diff
@@ -441,7 +441,7 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
 def to_json(msdf: MappingSetDataFrame) -> JsonObj:
     """Convert a mapping set dataframe to a JSON object."""
     doc = to_mapping_set_document(msdf)
-    data = JSONDumper().dumps(doc.mapping_set, contexts=doc.prefix_map)
+    data = JSONDumper().dumps(doc.mapping_set, contexts={"@context": doc.prefix_map})
```

Comment:
I think it's the case that none of the test data had ever had a properly loaded prefix map, so the fact that this wasn't working before was never caught by any tests

Reply:
Is this working now?

Reply:
Yes.

Reply:
Thanks!